.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  job_test_gem:
+    name: Test in source tree
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: windows
+            ruby: "head"
+          - os: windows
+            ruby: "2.5"
+          - os: ubuntu
+            ruby: "head"
+          - os: ubuntu
+            ruby: "3.2"
+          - os: ubuntu
+            ruby: "2.5"
```

Comment:
2.5 is old now ...

Reply:
if the tests runs sufficiently fast, we could include a lot more versions here...